site/en/userGuide/tools/attu_install-docker.md

```diff
@@ -17,20 +17,24 @@ This topic describes how to install Attu, an efficient open-source management to
 
 - Milvus installed on [your local device](install_standalone-docker.md) or [cluster](install_cluster-docker.md).
 - Docker 19.03 or later
+- Attu v2.1.0 or later
+- Milvus 2.1.0 or later
 
 <div class="alert note">
-Attu only supports Milvus 2.x.
+If useing Milvus 2.0.x , you can check <a href="https://milvus.io/docs/v2.0.x/attu_install-docker.md">v2.0.x attu doc</a>.
```

Comment:
See <a href="https://milvus.io/docs/v2.0.x/attu_install-docker.md">v2.0.x Attu doc</a> if you are using Milvus 2.0.x.